janet-http/http-service/src/main/java/io/techery/janet/HttpActionService.java

```diff
@@ -74,7 +75,8 @@ public HttpActionService(String baseUrl, HttpClient client, Converter converter)
         this.baseUrl = baseUrl;
         this.client = client;
         this.converter = converter;
-        this.runningActions = new CopyOnWriteArrayList<Object>();
+        this.actionHelperCache = new HashMap<Class, ActionHelper>();
+        this.runningRequests = new ConcurrentHashMap<Object, List<Request>>();
```

Comment:
yep. ok, I'm going to configure it.  
but it won't be large cache with many locks.